airdrop/airdrop.gno

```diff
@@ -44,8 +44,11 @@ type Airdrop struct {
 	snapshotManager *snapshot.Manager
 }
 
+var airdropToken grc20.Token
```

Comment:
Shouldn't this airdrop contract has to support multiple grc20 tokens?
If so, it can't be global variable

Reply:
I think this could supports single coin at a time. So I set it as global.